test/utils.ts

```
@@ -114,6 +115,19 @@ export function beeDebugUrl(url: string = beeUrl()): string {
   return urlObj.protocol + '//' + urlObj.hostname + ':' + port
 }
 
+/**
+ * Try to delete a chunk from local storage, ignoring all errors
+ *
+ * @param address  Swarm address of chunk
+ */
+export async function tryDeleteChunkFromLocalStorage(address: string): Promise<void> {
+  try {
+    await deleteChunkFromLocalStorage(beeDebugUrl(), address)
+  } catch (e) {
+    // ignore errors
```

I would be a bit cautious about ignoring all errors. For example, the Bee node might have turned off Debug API and this would not catch it (although then I guess will fail other tests as well). Could it be narrowed down to ignore only "not found" errors?